frontend/src/components/dashboard/LandingPage.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {makeStyles} from '@material-ui/core/styles';
+import BackgroundImage from './images/landing-page.JPG';
+import {Typography} from '@material-ui/core';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import ArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
+import {Link as ScrollLink} from 'react-scroll';
+
+const useStyles = makeStyles(() => ({
+  banner: {
+    background: 'url('+BackgroundImage+')',
+    backgroundPosition: 'center',
+    backgroundRepeat: 'no-repeat',
+    backgroundSize: 'cover',
+    height: '100vh',
+    margin: '0',
+    position: 'relative',
+    width: '100%',
+  },
+  bannerText: {
+    left: '30%',
+    position: 'absolute',
+    top: '40%',
+    transform: 'translate(-50%,-50%)',
+    width: '100%',
+    textAlign: 'center',
+  },
+}));
+
+export default function Banner() {
```

Comment:
Is this supposed to be `export default function LandingPage()` ? I could be mistaken

Reply:
it is, I'm not sure how it even worked properly with that bug